packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx

```diff
@@ -94,7 +102,7 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps) {
   };
 
   return (
-    <div className={styles.container}>
+    <div className={styles.container} ref={setMarkerElement}>
```

Comment:
I'm not entirely sure why we're passing a `useState` value in to a ref, shouldn't this be a `useRef` value?

Reply:
I was quite surprised too but this is what the [docs](https://popper.js.org/react-popper/v2/) says:
> Note: the usePopper hook intentionally takes the DOM node, not refs, in order to be able to update when the nodes change. A [callback ref](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs) is used here to permit this behaviour, and useState is an appropriate way to implement this.